### ISSUE TITLE
Failed to download package (Bitbucket)

### ISSUE THREAD

**Issue by Mergull** [dlang/dub-registry#392]
I can't download any package which use Bitbucket. I've tested on several DUB versions (1.12, 1.11, 1.9). So it's not a problem with DUB.

Example:
$dub fetch bluejay
Fetching bluejay 0.0.5...
Getting a release version failed: Failed to download package bluejay from https://dub-registry.herokuapp.com/packages/bluejay/0.0.5.zip
Retry with ~master...
Fetching bluejay ~master...
Failed to download package bluejay from https://dub-registry.herokuapp.com/packages/bluejay/~master.zip

**Comment by andre2007** [comment]
I just tried using dub 1.11 and were not able to reproduce the issue. Can you verify the issue is still reproducible for you? Maybe there was a temporary issue on bitbucket side?

**Comment by Mergull** [comment]
Ok. I tried on another computer (Linux Mint), and everything works fine. But I still have same issue on my computer (Linux Arch x86_64).
I noticed a problem about week ago. With --vverbose I get repetitive:

Fetching bluejay 0.0.5...
Acquiring package zip file
Downloading from 'https://code.dlang.org/packages/bluejay/0.0.5.zip'
Storing https://code.dlang.org/packages/bluejay/0.0.5.zip...
Download https://code.dlang.org/packages/bluejay/0.0.5.zip 302 Found (1.1)
Failed to download package bluejay from https://code.dlang.org/packages/bluejay/0.0.5.zip (Attempt 1 of 3)
